src/shared/functions.py

```diff
@@ -108,29 +112,34 @@ def update_metadata(
     if extra_items is None:
         extra_items = {}
     check_meta_type(meta_type)
+    logger.info(f"### Updating metadata {meta_type}")
+    logger.info(f"{study} {data_package} {version}")
+    logger.info(f"Key: {target} Value: {value}")
+    logger.info(f"Pre-update size: {len(metadata.keys)}")
```

Comment:
Did I actually catch this? well, we'll see. This logging is in here to hopefully create some breadcrumbs. Should be temporary.